gammapy/scripts/tests/test_make_bg_cube_models.py

```diff
@@ -0,0 +1,36 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from astropy.tests.helper import pytest, remote_data
+from ...datasets import get_path
+from ..make_bg_cube_models import main as make_bg_cube_models_main
+from ...datasets import make_test_dataset
+
+try:
+    import scipy
+    HAS_SCIPY = True
+except ImportError:
+    HAS_SCIPY = False
+
+
+@pytest.mark.skipif('not HAS_SCIPY')
+@pytest.mark.parametrize("extra_options,something_to_test", [
```

Comment:
I'd say leave it parametrised. It's parametrised now, and as you say, we definitely want to test at least a few common options in the future.

Concerning the issue that the test is slow: ignore that for now.

In the future we can make it faster, e.g. by reducing number of events and bins in the cube.
And the input test dataset can be shared across test runs, i.e. only be generated once per test session.
But improving the tests like that takes time, i.e. is future work.